### PR TITLE
Create symlink to <bfd/diagnostics.h> in bfd-headers

### PR DESCRIPTION
**Pull request by stilor**
Included by <bfd/bfd.h>.

Signed-off-by: Alexey Neyman <stilor@att.net>

### CODE REVIEW

**Comment by stilor** [comment]
Update the patch to also handle binutils 2.32... Is anyone alive here?

File: Makefile.in [hunk 1]
```diff
@@ -123,11 +123,16 @@ sinclude .deps
...
+# dangling.
 bfd-headers/.stamp:
 	rm -rf bfd-headers
 	mkdir bfd-headers
-	ln -sf $(BFD_INCLUDE_DIR)/bfd.h bfd-headers/bfd.h
-	for f in ansidecl filenames hashtab libiberty symcat; do \
+	for f in bfd bfd_stdint; do \
+		ln -sf $(BFD_INCLUDE_DIR)/$$f.h bfd-headers/$$f.h || exit 1; \
+	done
+	for f in ansidecl filenames hashtab libiberty symcat diagnostics; do \
```
Comment (vapier) [inline Makefile.in]: please keep this list sorted

**Comment by stilor** [comment]
Done.

**Comment by vapier** [comment]
thanks!

**Pull request merged**